infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -35,6 +35,11 @@ xrt_nightly_builds = [
 
 # Built on push to specific tag.
 versioned_builds = [
+  {
+    git_tag         = "v2.1.0"
+    package_version = "2.1"
+    accelerator     = "tpu"
+  },
```

Comment:
let's also add python 3.10 for the TPU build. User has been asking for this and we did that on nightly build too

Reply:
Updated